### PR TITLE
feat: [FX-1875] Add ApplicationUpdateNotification component

### PR DESCRIPTION
**Pull request by denieler**
[FX-1872]

### Description

Add `ApplicationUpdateNotification` component.

### Screenshots

Screenshot 2021-04-29 at 12 57 55

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

[FX-1872]: https://toptal-core.atlassian.net/browse/FX-1872

### CODE REVIEW

File: packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      onReloadClick,
+      onClose,
+      testIds,
+      ...rest
+    } = props
+    const classes = useStyles()
+
+    const notification = (
+      <>
+        <svg
```
Comment (vedrani) [inline packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx:45]: Why we don't have it as a separate component?

**Review by vedrani — COMMENTED** [review]
This component should be in topkit along with reload after deploy functionality, I don't see a reason to be in Picasso.

**Review by teimurjan — APPROVED** [review]
👍

File: packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx (hunk 1 above)
Comment (denieler) [reply packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx:45]: @vedrani my thoughts regarding this were the next: this component is a stateless UI component, which is reused in between many portals (Staff Portal, Talent Portal, etc.), no extra logic, no portal specific logic

File: packages/picasso/src/ApplicationUpdateNotification/styles.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    icon: {
+      opacity: 0.4,
+      position: 'absolute',
+      top: '-0.4em',
+      left: '-0.5em',
+      fill: palette.blue.darker
+    },
+    positionRelative: {
+      position: 'relative'
```
Comment (denieler) [inline packages/picasso/src/ApplicationUpdateNotification/styles.ts:26]: this is needed to send the reload icon behind the title and description

File: packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        </Container>
+
+        <Container top='small'>
+          <Button
+            variant='secondary'
+            onClick={onReloadClick}
+            data-testid={testIds?.reloadNowButton}
+          >
+            Reload Now
+          </Button>
```
Comment (denieler) [inline packages/picasso/src/ApplicationUpdateNotification/ApplicationUpdateNotification.tsx:76]: in the design currently, there is a secondary button with Main Blue color of the text, but we don't have such in Picasso. So Milos is considering now to add such button to BASE and meanwhile we decided to keep this button as just secondary from Picasso 👍

**Comment by polyankin-globaldots** [comment]
> This component should be in topkit along with reload after deploy functionality, I don't see a reason to be in Picasso.

Agree

**Review by vedrani — APPROVED** [review]
Approve, but I will push for Toptal strategy and vision based on our discussion on Friday.

**Pull request merged**